### DISCUSSION TITLE
How can I call V dll to C#

### DISCUSSION THREAD

**Discussion by luatdolphin** [vlang/v#19662] [Questions and Answers] [answered]
Can you help me with an example on how to use the shared library dll from V lang for a program developed in C#?

**Comment by JalonSolov** [comment]
C# can call C routines, so https://github.com/vlang/v/blob/master/doc/docs.md#calling-v-from-c

**Comment by Ddiidev** [comment, marked as answer]
I built a tool that creates a simple and automatic interface with a dll produced in V.
[SdkBuilder](https://github.com/ldedev/SdkBuilder)

Hope this works for you

**Reply by Wajinn** [reply]
Good idea.
